astronomy/sol_gravity_model.proto.txt

```diff
@@ -184,7 +252,7 @@ gravity_model {
     }
   }
   body {
-    # J2 coefficient and reference radius from MGN180U,
+    # Spherical harmonic coefficients and reference radius from MGN180U,
```

Comment:
`MGN180U` is a typo*; the model is called `MGNP180U` (note the URL below).

There is a proper citation for it: Konopliv, Banerdt, and Sjogren (1999), *Venus Gravity: 180th Degree and Order Model*.

The data can also be found at ftp://pds-geosciences.wustl.edu/mgn/mgn-v-rss-5-gravity-l2-v1/mg_5201/gravity/shgj180u.a01; this may be a more canonical source than a random GMAT fork on GitHub.

―
\* The typo does occur in the literature, in Kucinskas and Banerdt (1999), *Constraints on Venusian crustal and lithospheric properties from Magellan data*.